src/uct/sm/scopy/base/scopy_ep.c

```diff
@@ -68,30 +68,28 @@ uct_scopy_ep_prepare_tx(uct_ep_h tl_ep, const uct_iov_t *iov,
         }
 
         tx->iov[tx->iov_cnt] = iov[iov_it];
-        tx->total_length    += tx->iov[tx->iov_cnt].length;
+        total_length        += tx->iov[tx->iov_cnt].length;
```

Comment:
we use .length here, but uct_iov_get_length() in line 65, is that ok?
also can we avoid calculating total_length for release build?